pages/docs/setup/docker.mdx

```diff
@@ -37,16 +37,21 @@ import { Callout } from 'nextra-theme-docs'
         container_name: admin
         restart: unless-stopped
         environment:
-          ADMIN_GENERAL_HOST: 0.0.0.0
-          ADMIN_GENERAL_PORT: 7273
-          ADMIN_GENERAL_USERNAME: admin
-          ADMIN_GENERAL_PASSWORD: password
-          ADMIN_DRAGONITE_API_ENDPOINT: http://dragonite:7272
-          ADMIN_DRAGONITE_API_SECRET: secret
-          ADMIN_GOLBAT_API_ENDPOINT: http://golbat:9001
-          ADMIN_GOLBAT_API_SECRET: golbat1234
-        ports:
```

Comment:
Removing ports might be a bad idea, how should this be accessible from outside? :)